pkg/cmd/cli/cmd/create/route.go

```diff
@@ -113,7 +113,7 @@ func CreateEdgeRoute(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, ar
 		return err
 	}
 
-	wildcardpolicy := kcmdutil.GetFlagString(cmd, "wildcardpolicy")
+	wildcardpolicy := kcmdutil.GetFlagString(cmd, "wildcard-policy")
 	if len(wildcardpolicy) > 0 {
 		route.Spec.WildcardPolicy = api.WildcardPolicyType(wildcardpolicy)
 	}
```

Comment:
would it be worth checking to see if a user provided one of the supported wildcard-policy types? What happens if the flag is `--wildcard-policy="invalid"`?